package.json

```diff
@@ -57,6 +57,8 @@
   "main": "app.js",
   "private": true,
   "scripts": {
+    "docker:start": "docker-compose -f ./docker/docker-compose.yml up",
+    "docker:stop": "docker-compose -f ./docker/docker-compose.yml down",
```

Comment:
@seriouslysean What do you think about naming these commands `"start:docker"` and `"stop:docker"` to be more consistent with the other boot commands?

Reply:
I was originally going to do that, but we are going to end up with more docker commands and I'd prefer to colocate them under the docker namespace.